frontend/src/components/applications/StaffingCertificateInput.vue

```diff
@@ -0,0 +1,198 @@
+<template>
+  <v-form ref="form">
+    <v-row no-gutters class="mt-8">
+      <v-col v-for="(certificate, index) in certificates" :key="index" cols="12" sm="6" lg="4" xl="3" class="px-6 px-lg-8 mb-2" align="center">
+        <v-row no-gutters>
+          <v-col cols="12" sm="1" class="pt-2">{{ index + 1 }}.</v-col>
+          <v-col cols="12" sm="4" class="pt-1">
+            <v-text-field
+              v-model.trim="certificate.initials"
+              variant="outlined"
+              density="compact"
+              :disabled="readonly"
+              placeholder="Initials"
+              maxlength="4"
+              rounded="0"
+              :rules="rules.required"
+              :error-messages="getErrorMessagesForDuplicateError(certificate.isInitialsDuplicate)"
+              @input="update(certificate)"
+              @blur="checkDuplicateInitials" />
+          </v-col>
+          <v-col cols="12" sm="6" class="pt-1">
+            <v-text-field
+              v-model="certificate.certificateNumber"
+              variant="outlined"
+              density="compact"
+              :disabled="readonly"
+              :placeholder="certificateNumberPlaceholder"
+              maxlength="6"
+              rounded="0"
+              :rules="rules.required"
+              :error-messages="getErrorMessagesForDuplicateError(certificate.isCertificateNumberDuplicate)"
+              @input="update(certificate)"
+              @blur="checkDuplicateCertificateNumbers" />
+          </v-col>
+          <v-col v-if="certificate.initials || certificate.certificateNumber" cols="12" sm="1" class="pl-3" align="center">
+            <v-btn icon="mdi-trash-can-outline" variant="text" @click="remove(index)" />
+          </v-col>
+        </v-row>
+      </v-col>
+    </v-row>
+  </v-form>
+</template>
+
+<script>
+import { isEmpty } from 'lodash'
+import { useApplicationsStore } from '@/stores/applications'
+import { mapState } from 'pinia'
+import rules from '@/utils/rules'
+import { sanitizeWholeNumberInput } from '@/utils/common'
+import { APPLICATION_PROVIDER_EMPLOYEE_TYPES } from '@/utils/constants'
+
+export default {
+  name: 'StaffingCertificateInput',
+  props: {
+    readonly: {
+      type: Boolean,
+      default: false,
+    },
+    employeeCount: {
+      type: Number,
+      default: 0,
+    },
+    employeeType: {
+      type: Number,
+      required: true,
+    },
+    allCertificates: {
+      type: Array,
+      default: () => [],
+    },
+  },
+
+  emits: ['update:modelValue'],
```

Comment:
I think the use of v-model with an array is problematic as we see in the console.

Reply:
it's updated to use emit event instead of v-model. There is no more warning message. Thanks for your suggestions!!